projects/Allwinner/patches/linux/0063-HACK-SW-CEC-implementation-for-H3.patch

```diff
@@ -193,15 +193,5 @@ index b64d93da651d..e2936e7745b8 100644
 +	phy->bit_bang_cec = phy->disable_cec &&
 +			    !of_machine_is_compatible("roofull,beelink-x2");
  
- 	ret = of_address_to_resource(node, 0, &res);
- 	if (ret) {
-@@ -768,6 +844,9 @@ static int sun8i_hdmi_phy_remove(struct platform_device *pdev)
- {
- 	struct sun8i_hdmi_phy *phy = platform_get_drvdata(pdev);
- 
-+	cec_notifier_cec_adap_unregister(phy->cec_notifier, phy->cec_adapter);
-+	cec_unregister_adapter(phy->cec_adapter);
-+
```

Comment:
You should reinstate sun8i_hdmi_phy_remove(), but with only above 3 lines of code. It should be then referenced in sun8i_hdmi_phy_driver structure.

Reply:
Updated - https://github.com/LibreELEC/LibreELEC.tv/pull/6829/commits/38a99b09827e5759c034ec071073909a02dea78a